### ISSUE TITLE
Remove asyncio

### ISSUE THREAD

**Issue by link2xt** [adbenitez/simplebot_deltaland#3]
`deltachat_rpc_client` is not using asyncio anymore.

Once https://github.com/deltachat-bot/deltabot-cli-py/pull/2 is merged, this bot could remove asyncio everywhere.

**Comment by adbenitez** [comment]
here it is not only about asyncio lib but also all the sqlalchemy usage for async is a bit different from sync lib

**Comment by link2xt** [comment]
Yes, I have seen it, but all these libraries seem to support sync usage as well, so all asyncio can be removed.
